src/sections/About/style.module.scss

```diff
@@ -5,6 +5,11 @@
   flex-direction: column;
   gap: 6.25rem;
 
+  .isDesktop {
+    margin: 0 calc(-1 * vars.$side-padding);
+    width: 100vw;
```

Comment:
I'm getting horizontal scrollbars on Windows because of the `100vw`, can you try getting it working with `100%` instead?

![image](https://github.com/user-attachments/assets/e7526e01-73bd-4e54-88d8-361269563d27)